query_suggestions_v2/min_more.md

```diff
@@ -0,0 +1,7 @@
+Standard aggregation functions like [[queryFunctionName type=header text=min()]] interpolate data in each time series. The aggregation function  is applied to the interpolated series.  Aggregating thousands of time series affects query performance.
```

Comment:
I see double spaces between function and is and after the full stop.

The aggregation function  is applied to the interpolated series.  Aggregating thousands of time series affects query performance.